client/my-sites/upgrades/domain-management/components/header/index.jsx

```diff
@@ -11,7 +11,7 @@ const HeaderCake = require( 'components/header-cake' );
 const DomainManagementHeader = React.createClass( {
 	render() {
 		return (
-			<HeaderCake className="domain-management-header" onClick={ this.props.onClick }>
+			<HeaderCake className="domain-management-header" onClick={ this.props.onClick } backHref={ this.props.backHref }>
```

Comment:
Seems we have been using `onClick` for the `backHref` functionality? [example](https://github.com/Automattic/wp-calypso/blob/fcf789ebb43b7d8fe9a912d8fa82d5bee66573b5/client/my-sites/upgrades/domain-management/email-forwarding/index.jsx#L37) I like the `backHref` better. We should update the rest after this merge.